bigquery2/bigquery_constants.bal

```diff
@@ -16,7 +16,7 @@
 
 // API urls.
 const string BASE_URL = "https://www.googleapis.com";
-const string REFRESH_URL = "https://www.googleapis.com/oauth2/v3/token";
+public const string REFRESH_URL = "https://www.googleapis.com/oauth2/v3/token";
```

Comment:
Why are we making this public?

Reply:
With 0.990.3 version, we could be able to add refreshUrl into the user provided authConfig.
But with nightly 0.990.4-SNAPSHOT (Mar 25, 2019) I couldn't add this refreshUrl into authConfig.
That is why I changed to get this refreshUrl from the user.

https://github.com/wso2-ballerina/module-bigquery/blob/master/bigquery2/bigquery_endpoint.bal#L491

Reply:
This is because you are trying to set it to a union typed variable `ballerina/http:BasicAuthConfig|ballerina/http:OAuth2AuthConfig|ballerina/http:JwtAuthConfig` - https://github.com/ballerina-platform/ballerina-lang/blob/ef7c82b4d546b2752215b5b36a42d3442ddafe25/stdlib/http/src/main/ballerina/http/client_endpoint.bal#L328

You need to first assign authConfig.config to a variable, do the is check and then set refreshUrl.

Reply:
Fixed